api-server/common/models/user.json

```diff
@@ -172,6 +172,11 @@
       "description": "Camper is full stack certified (2018)",
       "default": false
     },
+    "isSciPyCert": {
+      "type": "boolean",
+      "description": "Camper is scientific computing with python certified",
+      "default": false
+    },
```

Comment:
I see additional certs in the code elsewhere:

```js
    isSciPyCert,
    isDataAnalysisPyCert,
    isMachineLearningCert
```

@freeCodeCamp/dev-team Did we miss adding any certs?

Reply:
Also, I am not a fan of server-side naming variations. For instance I see `scientificComputingPyId` at some places, and that relates to `isSciPyCert` presumably? We should keep these naming to be helpful for future maintenance.

Beau, Kris can you both let me know what should these be named correctly to? We have had much pain in the past because of chosen variables.

Reply:
Also did we miss the additional certs elsewhere in the code.

Reply:
Yes, I believe `scientificComputingPyId` should be paired with `isSciPyCert`. And it does seem like `api-server/common/models/user.json` is missing objects for the data analysis and machine learning certificates.

I do see what you mean about keeping things consistent. `isRespWebDesignCert` corresponds to `respWebDesignId`, `isInfosecQaCert` with `infosecQaId`, and so on.

Perhaps we can use the following scheme:
* Scientific Computing: `isSciCompPyCert` = `sciCompPyId`
* Data Analysis: `isDataAnalysisPyCert` = `dataAnalysisPyId`
* Machine Learning: `isMachineLearningPyCert` = `machineLearningPyId`

Reply:
That sounds much maintainable. @beaucarnes would that be ok?